envoy/USAGE.md

```diff
@@ -0,0 +1,25 @@
+## Purpose
```

Comment:
These 14 initial lines explain docker image building and thus should be (only) part of the README.md.

Reply:
README.md now states:

> An example implementation of oqs-enabled envoy terminating a tls handshake and proxying to an http backend has been included

Where has this been included? Link(s) would be helpful to understand this statement.